packages/common/.npmignore

```diff
@@ -0,0 +1,4 @@
+*
```

Comment:
any insights why did you decide to use allowed list here instead of disallowed list? 
I see one's struggles around it 😬 in the nearest future trying to figure out why git didn't `add` a new file

Reply:
😖oops, my bad, I was absolutely sure that this is a .gitignore file.